synfig-studio/src/gui/app.cpp

```diff
@@ -1303,83 +1303,84 @@ DEFINE_ACTION("keyframe-properties", _("Properties"));
 	}
 
 	// the toolbox
-	ACCEL("s",								"<Actions>/action_group_state_manager/state-normal"			);
-	ACCEL("m",								"<Actions>/action_group_state_manager/state-smooth_move"		);
-	ACCEL("l",								"<Actions>/action_group_state_manager/state-scale"			);
-	ACCEL("a",								"<Actions>/action_group_state_manager/state-rotate"			);
-	ACCEL("i",								"<Actions>/action_group_state_manager/state-mirror"			);
-	ACCEL("e",								"<Actions>/action_group_state_manager/state-circle"			);
-	ACCEL("r",								"<Actions>/action_group_state_manager/state-rectangle"			);
-	ACCEL("asterisk",								"<Actions>/action_group_state_manager/state-star"			);
-	ACCEL("g",								"<Actions>/action_group_state_manager/state-gradient"			);
-	ACCEL("o",								"<Actions>/action_group_state_manager/state-polygon"			);
-	ACCEL("b",								"<Actions>/action_group_state_manager/state-bline"			);
-	ACCEL("n",								"<Actions>/action_group_state_manager/state-bone"			);
-	ACCEL("t",								"<Actions>/action_group_state_manager/state-text"			);
-	ACCEL("u",								"<Actions>/action_group_state_manager/state-fill"			);
-	ACCEL("d",								"<Actions>/action_group_state_manager/state-eyedrop"			);
-	ACCEL("z",								"<Actions>/action_group_state_manager/state-zoom"			);
-	ACCEL("p",								"<Actions>/action_group_state_manager/state-draw"			);
-	ACCEL("k",								"<Actions>/action_group_state_manager/state-sketch"			);
-	ACCEL("w",								"<Actions>/action_group_state_manager/state-width"			);
+	ACCEL("s",				"<Actions>/action_group_state_manager/state-normal");
+	ACCEL("m",				"<Actions>/action_group_state_manager/state-smooth_move");
+	ACCEL("l",				"<Actions>/action_group_state_manager/state-scale");
+	ACCEL("a",				"<Actions>/action_group_state_manager/state-rotate");
+	ACCEL("i",				"<Actions>/action_group_state_manager/state-mirror");
+	ACCEL("e",				"<Actions>/action_group_state_manager/state-circle");
+	ACCEL("r",				"<Actions>/action_group_state_manager/state-rectangle");
+	ACCEL("asterisk",		"<Actions>/action_group_state_manager/state-star");
```

Comment:
We decided to use spaces for vertical adjustment. You can notice spacing above is broken for asterisk. That's because GitHub uses tab space of 8 width but on our IDE tab space is 4 so the below screenshot looks totally fine

![Screenshot from 2020-11-17 07-52-15](https://user-images.githubusercontent.com/24214509/99338447-3a253380-28aa-11eb-90b7-b4d279ed9c7e.png)

But that won't be the case if another user prefers tab space 2 or 8 (already seen in this PR)

Since you are changing all the vertical alignment indentation in this PR. I recommend you to update the changes again and use spaces instead.

Reply:
You're right.
I'll wait for #1848 to be merged to change spaces accordingly

Reply:
@FirasH done!

Reply:
@ice0 force pushed commit and cleaned spaces as well :+1:

Reply:
Resolved 👍🏽